app/build.gradle

```diff
@@ -41,6 +41,17 @@ dependencies {
     implementation 'com.google.firebase:firebase-ml-vision:24.0.0'
     // Face model
     implementation 'com.google.firebase:firebase-ml-vision-face-model:19.0.0'
+    implementation 'com.google.firebase:firebase-analytics:17.2.1'
+    implementation 'com.android.support:appcompat-v7:28.0.0'
+
+    implementation 'com.android.support.constraint:constraint-layout:1.0.2'
```

Comment:
Use `androidx.appcompat:appcompat:1.1.0' and `androidx.constraintlayout:constraintlayout:1.1.3` respectively instead of the support libraries. We want to keep the app size small and fast.

Reply:
resolved